### PR TITLE
Remove deprecated config_installer

### PR DESCRIPTION
**Pull request by piggito**
Module config_installer was deprecated and won't have a release for drupal 9 so we should replace it by core's site installer

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -83,7 +83,7 @@ TESTER_NAME = tester
 si:
 	@echo "Installing from: $(PROJECT_INSTALL)"
 ifeq ($(PROJECT_INSTALL), config)
-	$(call php, drush si config_installer --db-url=$(DB_URL) --account-name=$(ADMIN_NAME) --account-mail=$(ADMIN_MAIL) --account-pass=$(ADMIN_PW) -y config_installer_sync_configure_form.sync_directory=../config/sync)
+	$(call php, drush si --existing-config --db-url=$(DB_URL) --account-name=$(ADMIN_NAME) --account-mail=$(ADMIN_MAIL) --account-pass=$(ADMIN_PW) -y)
```
Comment (andypost) [inline Makefile:86]: Iirc it doesn't work with profile other then minimal

File: web/profiles/sdd/sdd.profile [hunk 2]
```diff
@@ -7,17 +7,65 @@
...
+  $config_factory = \Drupal::configFactory();
+
+  // Disable the user pictures on nodes.
+  $config_factory->getEditable('system.theme.global')
+    ->set('features.node_user_picture', FALSE)
+    ->save(TRUE);
+
+  // Disable field prefix.
+  $config_factory->getEditable('field_ui.settings')
+    ->set('field_prefix', '')
```
Comment (andypost) [inline web/profiles/sdd/sdd.profile:43]: Maybe put it into config?

File: Makefile (hunk 1 above)
Comment (alexkras) [reply Makefile:86]: there is also suggestion to use hook_install_tasks and may be help with https://www.drupal.org/project/drupal/issues/2982052 so we would be able to use own profile.

Comment (piggito) [reply Makefile:86]: I had the reference to issue https://www.drupal.org/project/drupal/issues/2982052 on code.
Also, I've tested it this way and it is working fine with custom profile

File: web/profiles/sdd/sdd.profile (hunk 2 above)
Comment (piggito) [reply web/profiles/sdd/sdd.profile]: Moved to config along with other settings and shortcut as default content

**Comment by andypost** [comment]
Just FYI https://www.drupal.org/project/drupal/issues/2982052 - hook could stay in place

File: Makefile (hunk 1 above)
Comment (piggito) [reply Makefile:86]: Confirmed in QA test

**Comment by davidferlay** [comment]
`drush si` returns this error https://gist.github.com/davidferlay/7a73bca15eb4ebaf15d7b42d3be69f78 @piggito

**Pull request merged**